package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "prebuild": "lerna bootstrap",
-    "build": "lerna run build --stream"
+    "build": "lerna run build --stream",
+    "build:all": "npm run prebuild && npm run build"
```

Comment:
nit - This shouldn't be necessary. npm-scripts will automatically run any `pre` or `post` scripts that match the current executing script. Did you try running `npm run build` in the root of the project? https://docs.npmjs.com/misc/scripts#description

Reply:
I did... and feel like if failed... but I could try again.